corehq/apps/reports/datatables/__init__.py

```diff
@@ -213,11 +211,13 @@ def render_html(self):
             elif isinstance(column, DataTablesColumnGroup):
                 groups.append(column.render_group_html)
             head.append(column.render_html)
-        head.append("</tr>")
-        groups.append("</tr>")
+
+        headerSegment = format_html_join('\n', '{}', ((segment,) for segment in head))
```

Comment:
OCD: 
```suggestion
        header_segment = format_html_join('\n', '{}', ((segment,) for segment in head))
```
(Don't let `aoColumns` drag you down. :upside_down_face: )